psutil/arch/osx/cpu.c

```diff
@@ -109,7 +114,52 @@ psutil_cpu_stats(PyObject *self, PyObject *args) {
     );
 }
 
+#if (defined(__arm64__) && defined(__APPLE__)) || defined(__aarch64__)
```

Comment:
defined(__APPLE__) seems redundant. This file is only loaded on macOS so it shouldn't be needed.